llvm/test/CodeGen/AArch64/GlobalISel/icmp-cst.ll

```diff
@@ -0,0 +1,185 @@
+; RUN: llc -mtriple=aarch64-linux-gnu -global-isel=1 < %s | FileCheck %s
```

Comment:
Please use update_llc_test_checks and add run lines for both SDAG and GISel.
```
; RUN: llc -mtriple=aarch64-linux-gnu -global-isel=0 < %s | FileCheck %s --check-prefixes=CHECK,CHECK-SD
; RUN: llc -mtriple=aarch64-linux-gnu -global-isel=1 < %s | FileCheck %s --check-prefixes=CHECK,CHECK-GI
```

Reply:
Done